app/src/main/java/com/smelnykov/homework_1/AuthActivity/AuthActivity.kt

```diff
@@ -56,18 +66,34 @@ class AuthActivity : AppCompatActivity() {
             }
 
             if (isAllFieldsCorrect) {
-                sendEmail(editTextEmail.text.toString())
+                registerAndChangeActivity(
+                    editTextEmail.text.toString(),
+                    editTextPassword.text.toString()
+                )
             }
         }
     }
 
     /**
-     * Method for sending email to My Profile Activity.
-     * @param email email which will be send
+     * Method for saving email, password and change activity to My Profile Activity.
+     * @param email email which will be saved
+     * @param password password which will be saved
+     * */
+    private fun registerAndChangeActivity(email: String, password: String) {
```

Comment:
two actions in one method, probably best split

Reply:
Thank you! Made changes. Commit 9e0bcee7